finder.py

```diff
@@ -7,20 +10,38 @@
 daddy = Daddy()
 
 
+
+
+
+
 @app.command()
 def check_domain(domain: str):
     daddy.print(f"{':white_heavy_check_mark:' if daddy.is_available(domain) else ':cross_mark:'} {domain} is {'NOT ' if not daddy.is_available(domain) else ''}available")
 
 
 @app.command()
-def search(desc: str, page_size=10):
+def search(desc: str, page_size: Optional[str] = 10,tlds: Optional[str]= '.com' '.ai' '.io'):
```

Comment:
1) Please fix page_size type
2) Turn tlds into a list of strings 

```suggestion
def search(desc: str, page_size: Optional[int] = 10, tlds: Optional[List[str]]=['.com' , '.ai', '.io']):
```

Do the same to `next` function